stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -43,5 +328,95 @@ public void hideHighlight(JsonRpcPeer peer, JSONObject params) {
 
     @JsonProperty(required = true)
     public String nodeValue;
+
+    @JsonProperty(required = false)
+    public Integer childNodeCount;
+
+    @JsonProperty(required = false)
+    public List<Node> children;
+
+    @JsonProperty(required = false)
+    public List<String> attributes;
+  }
+
+  public static enum NodeType {
+    ELEMENT_NODE(1),
+    TEXT_NODE(3),
+    PROCESSING_INSTRUCTION_NODE(7),
+    COMMENT_NODE(8),
+    DOCUMENT_NODE(9),
+    DOCUMENT_TYPE_NODE(10),
+    DOCUMENT_FRAGMENT_NODE(11);
+
+    private int mValue;
+
+    private NodeType(int value) {
+      mValue = value;
+    }
+
+    int getValue() {
```

Comment:
I was using an int in the Node for nodeType, but now I see the @JsonValue patter and have switched to it